docs/src/administration/users.md

```diff
@@ -1,65 +1,53 @@
-
-If you want your contributors to be able to see everything, but only commit to environments of a certain type, set their permission as **Project Viewer** for the whole project and update their permission on that environment type to **Contributor**.
-
-{{< note >}}
-The project owner - the person licensed to use Platform.sh - has the project administrator role. The project owner is the only user that can make changes to the project plan. 
-{{< /note >}}
-
-## SSH access
-
-By default, everyone with access equal or greater than **Contributor** can access the project through SSH. 
-
-To customize who can use SSH, [set the `access` key](/configuration/app/access.md) in your `.platform.app.yaml` file.
-
-## Manage user permissions in the console
-
-From your list of projects, select the project where you want to view or edit user permissions. Click the Settings tab at the top of the page, then click the `Access` tab on the left to see the users in the project and their roles.
-
-![Project user management screenshot](/images/management-console/settings-project-access.png)
+* Environments can share the same type.
+* Environments can change their type from Staging to Development and the other way around.
+* After you add or remove a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command platform redeploy or the Redeploy button in the management console.
 
-Selecting a user to edit that user's permissions or delete the user's access to the project entirely.
 
-To add a new user, click the `Add` button.
 
-You can either grant the `Project admin` role to the user, which will give them `Admin` access to every environment in the project, or grant specific permissions for each environment type.
+| Role                     | Description |
+|------------------------- |-------------|
+|Viewer                    | Viewers can view all environments of this type.|
+|Contributor               | This role includes the viewer permissions. Contributors can push code and branch all environments of this type. By default, everyone with access equal or greater than Contributor can access the environment through SSH. To customize who can use SSH, [set the access key](https://docs.platform.sh/configuration/app/access.html) in your platform.app.yaml |
+|Admin                     | This role includes viewer and contributor permissions. Administrators can change settings and execute actions on all environments of this type. By default, everyone with access equal or greater than Contributor can access the environment through SSH. To customize who can use SSH [set the access key](https://docs.platform.sh/configuration/app/access.html) in your platform.app.yaml|
 
-After inviting a new user, users without a Platform.sh account receive an invitation email asking to confirm their details and register an account.
+{{< note theme="warning" title="Warning" >}}
```

Comment:
```suggestion
{{< note theme="warning" >}}
```

It's given the title "Warning" by default. It's not a great title, but no need to specify it. :slightly_smiling_face: